openlane/steps/step.py

```diff
@@ -1040,26 +1159,17 @@ def run_subprocess(
         silent: bool = False,
         report_dir: Optional[Union[str, os.PathLike]] = None,
         env: Optional[Dict[str, Any]] = None,
+        *,
+        check: bool = True,
+        output_processing: Optional[Sequence[Type[OutputProcessor]]] = None,
         _popen_callable: Callable[..., psutil.Popen] = psutil.Popen,
         **kwargs,
     ) -> Dict[str, Any]:
```

Comment:
Since we are breaking the API, I think we should return an object similar to Python's `subprocess.run`.

Reply:
I don't see the value entirely. Given the dynamic nature of the object it would more or less be syntactic sugar that would just make the type-checker angry.